### ISSUE TITLE
amax to ONNX opset version 12 is not supported

### ISSUE THREAD

**Issue by conceptofmind** [pytorch/pytorch#75598]
### 🐛 Describe the bug

Amax operator used in model architecture:

`sim = sim - sim.amax(dim=-1, keepdim=True).detach()`

Attempt to export Onnx/PyTorch model:

```
input_names = ['Input']
output_names = ['Output']
tokens = torch.randint(0, 20000, (1, 2048))
torch.onnx.export(model, tokens, 'model.onnx', input_names=input_names, output_names=output_names, opset_version=12)
```

Error:

RuntimeError: Exporting the operator amax to ONNX opset version 12 is not supported. Please feel free to request support or submit a pull request on PyTorch GitHub.

### Versions

Collecting environment information...
PyTorch version: 1.11.0
Is debug build: False
CUDA used to build PyTorch: 11.3
ROCM used to build PyTorch: N/A

OS: Ubuntu 18.04.6 LTS (x86_64)
GCC version: (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
Clang version: Could not collect
CMake version: version 3.22.3
Libc version: glibc-2.27

Python version: 3.9.12 (main, Apr  5 2022, 06:56:58)  [GCC 7.5.0] (64-bit runtime)
Python platform: Linux-5.4.0-107-generic-x86_64-with-glibc2.27
Is CUDA available: True
CUDA runtime version: 11.3.109
GPU models and configuration: GPU 0: NVIDIA GeForce RTX 3090
Nvidia driver version: 465.19.01
cuDNN version: Probably one of the following:
/usr/local/cuda-11.3/targets/x86_64-linux/lib/libcudnn.so.8.2.1
/usr/local/cuda-11.3/targets/x86_64-linux/lib/libcudnn_adv_infer.so.8.2.1
/usr/local/cuda-11.3/targets/x86_64-linux/lib/libcudnn_adv_train.so.8.2.1
/usr/local/cuda-11.3/targets/x86_64-linux/lib/libcudnn_cnn_infer.so.8.2.1
/usr/local/cuda-11.3/targets/x86_64-linux/lib/libcudnn_cnn_train.so.8.2.1
/usr/local/cuda-11.3/targets/x86_64-linux/lib/libcudnn_ops_infer.so.8.2.1
/usr/local/cuda-11.3/targets/x86_64-linux/lib/libcudnn_ops_train.so.8.2.1
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

Versions of relevant libraries:
[pip3] numpy==1.22.3
[pip3] PaLM-pytorch==0.0.12
[pip3] torch==1.11.0
[pip3] torchaudio==0.11.0
[pip3] torchvision==0.12.0
[conda] blas                      1.0                         mkl  
[conda] cudatoolkit               11.3.1               h2bc3f7f_2  
[conda] ffmpeg                    4.3                  hf484d3e_0    pytorch
[conda] mkl                       2021.4.0           h06a4308_640  
[conda] mkl-service               2.4.0            py39h7f8727e_0  
[conda] mkl_fft                   1.3.1            py39hd3c417c_0  
[conda] mkl_random                1.2.2            py39h51133e4_0  
[conda] numpy                     1.22.3                   pypi_0    pypi
[conda] numpy-base                1.21.2           py39h79a1101_0  
[conda] palm-pytorch              0.0.12                   pypi_0    pypi
[conda] pytorch                   1.11.0          py3.9_cuda11.3_cudnn8.2.0_0    pytorch
[conda] pytorch-mutex             1.0                        cuda    pytorch
[conda] torchaudio                0.11.0               py39_cu113    pytorch
[conda] torchvision               0.12.0               py39_cu113    pytorch

**Comment by fatcat-z** [comment]
The PR https://github.com/pytorch/pytorch/pull/75268 has resolved this issue. Please try a nightly build or wait for the next version of PyTorch to see if your problem has gone.
Thanks.

**Comment by MohamedAliRashad** [comment]
@fatcat-z 
Will this be available in torch 1.12 ?

**Comment by fatcat-z** [comment]
> @fatcat-z Jay Zhang FTE Will this be available in torch 1.12 ?

Yes, I believe so.

**Comment by BowenBao** [comment]
Closing as supported since 1.12.
